_search-plugins/sql/full-text.md

```diff
@@ -119,13 +119,13 @@ To search for text in multiple fields, use `MULTI_MATCH` function. This function
 
 ### Syntax
 
-The `MULTI_MATCH` function lets you *boost* certain fields using **^** character. Boosts are multipliers that weigh matches in one field more heavily than matches in other fields. The syntax allows to specify the fields in double quotes, single quotes,  surrounded by backticks, or unquoted. Use star ``"*"`` to search all fields. Star symbol should be quoted.
+The `MULTI_MATCH` function lets you *boost* certain fields using **^** character. Boosts are multipliers that weigh matches in one field more heavily than matches in other fields. The syntax allows to specify the fields in double quotes, single quotes, surrounded by backticks, or unquoted. Use star ``"*"`` to search all fields. Star symbol should be quoted.
 
 ```sql
 multi_match([field_expression+], query_expression[, option=<option_value>]*)
 ```
 
-The weight is optional and is specified after the field name. It could be delimited by the `caret` character -- `^` or by whitespace. Please, refer to examples below:
+The weight is optional and is specified after the field name. It could be delimited by the `caret` character -- `^` or by whitespace. Refer to the following examples:
```

Comment:
```suggestion
The weight is optional and is specified after the field name. It could be delimited by the `caret` character -- `^` or by white space. Refer to the following examples:
```